chapters/annotations.tex

```diff
@@ -1432,39 +1365,60 @@ \section{Annotations for the Graphical User Interface}\label{annotations-for-the
 Setting \lstinline!filter! only shows files that fulfill the given pattern defined by \lstinline!text1 (*.ext1);;text2 (*.ext2)! to show only files with file extension \filename{ext1} or \filename{ext2} and displaying a description text \lstinline!text1! and \lstinline!text2!, respectively.
 Parameter \lstinline!caption! is the text displayed in the dialog menu.
 Parameter \lstinline!loadSelector! is used to select an existing file for reading, whereas parameter \lstinline!saveSelector! is used to define a file for writing.
 
+The \lstinline!groupImage! references an image using an URI (see \cref{external-resources}), and the image is intended to be shown together with the parameter-group (only one image per group is supported).
+Disabling the input field will not disable the image.
+The background of the \lstinline!groupImage! and any image used in HTML-documentation is recommended to be transparent (intended to be a light color) or white.
+
+The \lstinline!connectorSizing! is described separately in \cref{connector-sizing}.
+
 \begin{example}
 \begin{lstlisting}[language=modelica]
 model DialogDemo
   parameter Boolean b = true "Boolean parameter";
   parameter Modelica.Units.SI.Length length "Real parameter with unit";
-  parameter Integer nInports=0
-     annotation(Dialog(connectorSizing=true));
+  parameter Integer nInports = 0
+     annotation(Dialog(connectorSizing = true));
   parameter Real r1 "Real parameter in Group 1"
-     annotation(Dialog(group="Group 1"));
+     annotation(Dialog(group = "Group 1"));
   parameter Real r2 "Disabled Real parameter in Group 1"
-     annotation(Dialog(group="Group 1",enable = not b));
+     annotation(Dialog(group = "Group 1", enable = not b));
   parameter Real r3 "Real parameter in Tab 1"
-     annotation(Dialog(tab="Tab 1"));
+     annotation(Dialog(tab = "Tab 1"));
   parameter Real r4 "Real parameter in Tab 1 and Group 2"
-     annotation(Dialog(tab="Tab 1", group="Group 2"));
+     annotation(Dialog(tab = "Tab 1", group = "Group 2"));
   StepIn stepIn[nInports];
   ...
 end DialogDemo;
 \end{lstlisting}
-When clicking on an instance of model \lstinline!DialogDemo!, a menu pops up
-that may have the following layout (other layouts are also possible,
-this is vendor specific). Note, parameter \lstinline!nInports! is not present in the
-menu since it has the \lstinline!connectorSizing! annotation and therefore it
-should not be modified by the user (an alternative is to show parameter
-\lstinline!nInports! in the menu but with disabled input field):
+When clicking on an instance of model \lstinline!DialogDemo!, a menu pops up that may have the following layout (other layouts are also possible, this is vendor specific).
+Note, parameter \lstinline!nInports! is not present in the menu since it has \lstinline!connectorSizing = true! and therefore it should not be modified by the user (an alternative is to show parameter \lstinline!nInports! in the menu but with disabled input field):
```

Comment:
Note that this is now a forward reference, as connectorSizing isn't yet defined.

Reply:
Did you notice it comes after the forward reference was made explicit here?
> The \lstinline!connectorSizing! is described separately in \cref{connector-sizing}.

If so, then how about this:
```suggestion
Note, parameter \lstinline!nInports! is not present in the menu since it has \lstinline!connectorSizing = true! (see \cref{connector-sizing}) and therefore it should not be modified by the user (an alternative is to show parameter \lstinline!nInports! in the menu but with disabled input field):
```

Reply:
Alternatively, you could remove the mention inPorts from the example. The images would be the same. And then add a sentence in the connector-sizing section, after lines 1449-1451 :
>If \lstinline!connectorSizing = true!, a tool may set the parameter value
in a modifier automatically, if used as dimension size of a vector of
connectors.

saying something like:
> In that case the parameter should not be modified by the user, and a tool may choose to not display that parameter in the menu or display it with disabled input field.

Reply:
@eshmoylova this also sounds good to me.  Now I just want to find the resolution that makes the most reviewers happy.
@HansOlsson, how about @eshmoylova's suggestion?

Reply:
Would be ok.

Reply:
Done according to @eshmoylova's suggestion, except that I called it a _dialog_ rather than a _menu_.  See also separate comment below regarding _menu_.

Reply:
Looks good to me. Resolving.